src/PeakFinder.hpp

```diff
@@ -0,0 +1,86 @@
+#ifndef SHASTA_PEAKFINDER_HPP
```

Comment:
Consistently with other Shasta `.hpp` files, please add an underscore after `PEAK`, as we use underscores to separate words in include guards.